_includes/releases/v22.2/v22.2.1.md

```diff
@@ -0,0 +1,386 @@
+This bug was present since version v22.1.0. [#89250][#89250]
+- When writing storage checkpoints on consistency checker failures, flushing WAL was disabled, so some checkpoints could be slightly out of date. This is now fixed. [#89403][#89403]
+- This patch fixes [optimizer](../v22.2/cost-based-optimizer.html) selectivity and cost estimates of zigzag joins in order to prevent query plans from using it when it would perform poorly (when many rows are qualified). [#89427][#89427]
+- Updated hot ranges, problem ranges, data distribution, stores report, range status, raft for all ranges features to require `VIEWCLUSTERMETADATA`. Updated the [`SHOW CLUSTER SETTING`](../v22.2/show-cluster-setting.html) SQL command to require `VIEWCLUSTERSETTING/MODIFYCLUSTERSETTING` privileges. Fixed a visual bug on stores report where an error shows an infinite spinner only. [#89508][#89508]
+- Fixed a bug introduced in v20.2 that could cause filters to be dropped from a query plan with many joins in rare cases. [#89158][#89158]
+- Narrowed down the conditions under which a `VOTER_DEMOTING_LEARNER` can acquire the lease in a joint configuration to:
+    a) There has to be a `VOTER_INCOMING` in the configuration, and 
+    b) The `VOTER_DEMOTING_LEARNER` was the last leaseholder. 
+This prevents it from acquiring the lease unless it is the only one that can acquire it. Transferring the lease away is necessary before exiting the joint configuration (without the fix the system can be stuck in a joint configuration in some rare situations). [#89594][#89594]
+- Fixed tables created by [userfile storage](../v22.2/use-userfile-for-bulk-operations.html) that have invalid foreign key constraints. [#89371][#89371]
+- Fixed a crash that could occur when dropping a role that owned two schemas with the same name in different databases. The bug was introduced in v22.1.0. [#89534][#89534]
+- Excluded check constraints of hash-sharded indexes from being unvalidated when executing [`IMPORT INTO`](../v22.2/import-into.html). [#89525][#89525]
+- Avoided a source of internal connectivity problems that would resolve after restarting the affected node. [#89597][#89597]
+- Previously, uncommitted privileges could be cached if a transaction is rolled back. This is now fixed. This bug was only present in the v22.2 alpha and beta versions. Example:
+
+    ~~~sql
+    BEGIN; 
+    GRANT SELECT ON crdb_internal.tables TO testuser; 
+    SELECT has_table_privilege('testuser', 'crdb_internal.tables', 'SELECT'); --- this caches the privilege --- 
+    ROLLBACK; --- SELECT IS STILL CACHED UNTIL ANOTHER GRANT/REVOKE HAPPENS TO INVALIDATE THE CACHE--- 
+    ~~~ 
+
+[#89717][#89717]
```

Comment:
nit: I would move this to immediately following `Example:` (same line) to avoid this link consuming a line on its own. Looks a bit off both ways, but less so interrupting `Example:` flow than "next-entry" flow, imo. Feel free to disregard if you disagree!